### DISCUSSION TITLE
I cant add bug report any more ...everytime it (bot) says template issue

### DISCUSSION THREAD

**Discussion by J-khatri** [bromite/bromite#1124] [Q&A]
this is funny,
Now I'm not able to submit any but, even if i do not edit or change anything I just reply yes or no in the bug report !!!

is this a known issue ?

**Comment by csagan5** [comment]
You have to complete the bug report. Please do not submit tens of bug reports every time you try. Submit only one and make sure you complete all parts of it.

Read again each paragraph of the report and you will see what you are missing; you can edit the same bug report without re-submitting it.

**Reply by jobnoorman** [reply]
Looking at the [list](https://github.com/bromite/bromite/issues?q=label%3Amissing-issue-template+) of issues that got closed due to `missing-issue-template`, I found a few that seem to correctly follow the template. Could there be something wrong with the bot? If not, it might be helpful to give some hint about exactly what is wrong because even after triple checking #1129, I have no idea how to make it pass the test :-)

**Reply by csagan5** [reply]
They seem, but they are not. #1129 is not using the latest version of Bromite; the bot is not perfect but right now I have not had time to make it more verbose. It gives the correct message when you create a new issue and the version is incorrect but not when you make edits.

**Reply by jobnoorman** [reply]
I see, so what am I supposed to fill-in if I cannot easily test the latest version? Previously I said something like "Cannot test" but this wasn't accepted either. I don't want to say "no" because I'm pretty sure it will be reproducible.

Alternatively, is there a way to run Bromite's SystemWebView inside an emulator? If so, I can just verify the bug also occurs in the latest version.

**Reply by csagan5** [reply]
I suggest to try reproducing it with the regular browser instead of the webview.

**Reply by jobnoorman** [reply]
I run into this bug in an app that uses a WebView via [capacitor](https://capacitorjs.com/) so I don't think it is possible to run it in a regular browser.

**Reply by csagan5** [reply]
@JobNoorman webviews are not supported anyways, they are provided "as is"; if you cannot reproduce the issue in a regular browser there will be no way for us to reproduce it.
